src/beamlime/applications/_nexus_helpers.py

```diff
@@ -39,23 +39,25 @@ def _merge_ev44(group, message):
         dataset = _init_dataset(group, field, value)
         if field in ('time_of_flight', 'pixel_id'):
             dataset['config']['values'] = np.concatenate(
-                (dataset['config']['values'], value)
+                (dataset['config']['values'], np.asarray(value))
```

Comment:
When does the absence of `np.asarray` cause an issue here?

Reply:
Hmm actually it doesn't matter here since concatenate will make it into numpy ndarray anyways.